earth_enterprise/rpms/build.gradle

```diff
@@ -43,20 +43,6 @@ apply plugin: "nebula.ospackage"
 
 def rpmPlatformString = com.tsciences.os.Platform.rpmPlatformString
```

Comment:
I tested the rpm on centos 7 to verify that it would upgrade my postgreSQL databases and it did. The run searchs returned the same results as before the upgrade using the rpm but i had one issue. When the databases were being restored from dumps, i get errors/warning about "plpgsql":

```Restoring geserve-databases from dump /var/opt/google/pgsql/data.backup_dump.26393/ ...
Adding PostGIS extension
CREATE EXTENSION
Restoring geendsnippet.dump...
Restoring gepoi.gepoi_1.dump...
Restoring gesearch.dump...
 setval 
--------
      1
(1 row)

 setval 
--------
      1
(1 row)

 setval 
--------
      6
(1 row)

Restoring gestream.dump...
 setval 
--------
      1
(1 row)

 setval 
--------
      1
(1 row)

 setval 
--------
     29
(1 row)

 setval 
--------
      4
(1 row)

 setval 
--------
      4
(1 row)

 setval 
--------
      3
(1 row)

psql:/opt/google/share/opengee-server/sql/gesearch_tables.sql:20: ERROR:  language "plpgsql" already exists
 create_tables 
---------------
 
(1 row)

 run_searchdef_upsert 
----------------------
 
(1 row)

psql:/opt/google/share/opengee-server/sql/gesearch_tables.sql:167: ERROR:  cannot drop language plpgsql because extension plpgsql requires it
HINT:  You can drop extension plpgsql instead.
psql:/opt/google/share/opengee-server/sql/gestream_tables.sql:20: ERROR:  language "plpgsql" already exists
 create_or_update_tables 
-------------------------
 
(1 row)

psql:/opt/google/share/opengee-server/sql/gestream_tables.sql:147: ERROR:  cannot drop language plpgsql because extension plpgsql requires it
HINT:  You can drop extension plpgsql instead.
Done.
waiting for server to shut down.... done
server stopped
upgrade done
The PostgreSQL component is successfully installed.```

Reply:
I think it's an incompatibility between dumps created in 8.4 and restored in 9.6. As far as I've been able to tell they don't cause any problems. I'm afraid that trying to fix it so it doesn't print errors could cause more issues since we'd have to modify the dump files. So I'm recommending just living with the errors.

Reply:
Yes, indeed, those errors do not seem to cause any issues as far as databases restoration goes. They seem to be  caused by change how postgreSQL does treat 'language "plpgsql"'. In recent versions of postgreSQL (starting in 9.0) ""plpgsql" is installed by default, so we don't need to try to install or drop it in our '.../src/server/gesearch_tables.sql' and '.../src/server/gestream_tables.sql' any more since we are using postgreSQL 9.6.5.

For reference: https://www.postgresql.org/docs/current/static/release-9-0.html
See " PL/pgSQL is now installed by default, ...".

I made local changes in both mentioned .sql files, re-built the rpm and did an install. Those errors were not occurring any more, my databases were restored ok and the search results returned same results as before rpm upgrade execution.

Here are the needed changes:
- Delete or comment out lines 20 and 167 in '../src/server/gesearch_tables.sql'
- Delete or comment out lines 20 and 147 in '../src/server/gestream_tables.sql'

Reply:
Removed those lines as instructed and it does fix the errors. I didn't realize those sql files were still be run on the upgrade, but that's good to know :)

Reply:
For re-verification i rebuilt the RPM and executed the installers. I did not see the errors any more and my functional tests for the published databases and search worked just fine. 

I noticed however that we always dump and restore the databases even if we are dealing with the same PostgreSQL version (9.6.5 --> 9.6.5). This is fine for now. We can add some smart into it, eventually later, to skip the dump/restore if the same version of PostgreSQL is already installed.
This can be done in a later ticket.